homeassistant/components/rflink/cover.py

```diff
@@ -142,17 +142,17 @@ def _handle_event(self, event):
             self._state = False
 
     @property
-    def should_poll(self):
+    def should_poll(self) -> bool:
         """No polling available in RFlink cover."""
         return False
 
     @property
-    def is_closed(self):
+    def is_closed(self) -> bool | None:
         """Return if the cover is closed."""
         return not self._state
 
     @property
-    def assumed_state(self):
+    def assumed_state(self) -> bool:
```

Comment:
Same

Reply:
This one I cannot do, because the assumed_state is already overriden in the base `RflinkDevice`
https://github.com/home-assistant/core/blob/90c68085bef5b91e9f15fb11f4eb9fc83072c1fc/homeassistant/components/rflink/__init__.py#L389-L392